### PR TITLE
add details

### PR DESCRIPTION
**Pull request by ashishmangla0**

### CODE REVIEW

File: members/ashish/data.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+	"id": "sumitdhanania",
```
Comment (ankushdharkar) [inline members/ashish/data.json:2]: ID is wrong

Comment (ashishmangla0) [reply members/ashish/data.json:2]: @ankushdharkar  my id is right. https://github.com/ashishmangla0  is mu github url

**Comment by Harshi7016** [comment]
Shouldn't Id be ashish here as you have named your folder as ashish and in data.json you have mentione d sumit...

File: members/ashish/data.json (hunk 1 above)
Comment (ankushdharkar) [reply members/ashish/data.json:2]: @ashishmangla0 But you have added the id "sumitdhanania". Please check.

Comment (ashishmangla0) [reply members/ashish/data.json]: @ankushdharkar  my bad. make changes.

**Comment by ashishmangla0** [comment]
created by mistake

**Pull request closed without merging**